Dockerfile

```diff
@@ -30,9 +30,10 @@ RUN openssl req \
     -days 365 \
     -nodes \
     -x509 \
-    -subj "/C=US/ST=Denial/L=Springfield/O=Dis/CN=www.example.com" \
+    -subj "/C=US/ST=Denial/L=Springfield/O=Dis/CN=localhost" \
     -keyout privkey.pem \
     -out fullchain.pem
 
 EXPOSE 8443
-CMD ["node", "./bin/solid", "start"]
+ENV NODE_TLS_REJECT_UNAUTHORIZED=0
```

Comment:
@dmitrizagidulin I don't think this is good to have as a default in the Dockerfile, as it opts-into allowing requests that might have a man-in-the-middle, and I don't think that's a good default for production deployments.

Even without this in here, it's still possible for someone to opt into the insecure option with `docker run -e NODE_TLS_REJECT_UNAUTHORIZED=0 interopalliance/life-server`

Reply:
also this works:

```
docker run life-server bin/solid-test start
```

Reply:
You are absolutely right; I’ll change it.